app/helpers/application_helper/button/transform_vm_button.rb

```diff
@@ -0,0 +1,12 @@
+class ApplicationHelper::Button::TransformVmButton < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.vendor == "vmware"
+  end
+
+  def disabled?
+    # Is there a provider that supports import?
+    EmsInfra.all.select(&:validate_import_vm).empty?
+  end
```

Comment:
@martinpovolny well we are not adding anything new, this has been long part of the backend for the requirements of the API - which exactly uses `validates_<action_name>` to show/hide availiability of an action, so I'm just reusing this existing logic (note this was implemented before the 'feature supported' infra was added)